src/System.Globalization/tests/CultureInfo/CultureInfoCurrentCulture.cs

```diff
@@ -57,6 +57,7 @@ public void CurrentUICulture()
         }
 
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap, "Thread cultures is not honored in UWP.")]
```

Comment:
Using this API is OK. it is just kind of no-op in UWP. but it is important in no UWP cases.